src/deployers/PlatformPluginsDeployer.cpp

```diff
@@ -17,8 +17,23 @@ bool PlatformPluginsDeployer::deploy() {
 
     ldLog() << "Deploying platform plugins" << std::endl;
 
-    if (!appDir.deployLibrary(qtPluginsPath / "platforms/libqxcb.so", appDir.path() / "usr/plugins/platforms/"))
-        return false;
+    if (getenv("ENABLE_EGLFS")){
+        if (!appDir.deployLibrary(qtPluginsPath / "platforms/libqxcb.so", appDir.path() / "usr/plugins/platforms/"))
+            ldLog() << "No XCB platform plugin found, skipping" << std::endl;
+        if (!appDir.deployLibrary(qtPluginsPath / "platforms/libqeglfs.so", appDir.path() / "usr/plugins/platforms/"))
+            ldLog() << "No EGLFS platform plugin found, skipping" << std::endl;
+        if (!appDir.deployLibrary(qtPluginsPath / "egldeviceintegrations/libqeglfs-kms-integration.so", appDir.path() / "usr/plugins/egldeviceintegrations/"))
+            ldLog() << "No EGLFS KMS Integration plugin found, skipping" << std::endl;
+        if (!appDir.deployLibrary(qtPluginsPath / "egldeviceintegrations/libqeglfs-x11-integration.so", appDir.path() / "usr/plugins/egldeviceintegrations/"))
+            ldLog() << "No EGLFS X11 Integration plugin found, skipping" << std::endl;
+        if (!appDir.deployLibrary(qtPluginsPath / "egldeviceintegrations/libqeglfs-kms-egldevice-integration.so", appDir.path() / "usr/plugins/egldeviceintegrations/"))
+            ldLog() << "No EGLFS KMS EGLDEVICE Integration plugin found, skipping" << std::endl;
+        if (!appDir.deployLibrary(qtPluginsPath / "egldeviceintegrations/libqeglfs-kms-emu-integration.so", appDir.path() / "usr/plugins/egldeviceintegrations/"))
+            ldLog() << "No EGLFS EMU Integration plugin found, skipping" << std::endl;
+    } else {
+        if (!appDir.deployLibrary(qtPluginsPath / "platforms/libqxcb.so", appDir.path() / "usr/plugins/platforms/"))
```

Comment:
The term "enable" implies that enabling the env var just adds additional files. But that's not true: it is more like "either egl or xcb". The name or at least the docs should reflect that. I think `USE_` would be a better prefix in the current state.

Can xcb be deployed safely next to egl?

Reply:
yes, by default they're both installed on Debian, you select the platform plugin with --plugin flag or an environment variable